Orange/tests/test_color.py

```diff
@@ -9,21 +9,19 @@
 
 
 class GradientPaletteGeneratorTest(unittest.TestCase):
+
     def test_two_color(self):
         generator = GradientPaletteGenerator('#000', '#fff')
-        np.testing.assert_equal(generator.getRGB(.5), (128, 128, 128))
-        np.testing.assert_equal(generator.getRGB(np.nan), NAN_GREY)
-        np.testing.assert_equal(generator.getRGB([0, .5, np.nan]),
-                                [(0, 0, 0),
-                                 (128, 128, 128),
-                                 NAN_GREY])
+        for arg in ((.5, (128, 128, 128)),
```

Comment:
Instead of `arg[0]` and `arg[1]` below, use:
```py
for float_values, rgb in ...
```
Please perform this kind of rationalized unpacking everywhere where applicable.